packages/frontend-shared/src/testing/commands.ts

```diff
@@ -0,0 +1,231 @@
+import '@testing-library/cypress/add-commands'
+import { mount, CyMountOptions } from '@cypress/vue'
+import urql, { TypedDocumentNode, useQuery } from '@urql/vue'
+import { print, FragmentDefinitionNode } from 'graphql'
+import { testUrqlClient } from './testUrqlClient'
+import { Component, computed, defineComponent, h } from 'vue'
+
+import { ClientTestContext } from './ClientTestContext'
+import type { TestSourceTypeLookup } from '@packages/graphql/src/testing/testUnionType'
+import { createI18n } from '@packages/launchpad/src/locales/i18n'
```

Comment:
I am not sure how to work around this - ideally we need some way to inject the translations, but it's not really practical with how Cypress custom commands are declared. We need some way to know if the mount function is getting used in `app` or `launchpad` and conditionally require the translations.